tools/check_pragma_once.py

```
@@ -35,12 +35,7 @@
             has_pragma = True
             break
         else:
-            if not (
-                l.startswith(r"//")
-                or l.startswith(r"/*")
-                or l.startswith(r"/*")
-                or l.startswith("\n")
-            ):
+            if not l.startswith((r"//", r"/*", r"/*", "\n")):
```

Ruff (0.16.0)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)